### ISSUE TITLE
The fine line between people and ideas

### ISSUE THREAD

**Issue by mnot** [linuxwolf/ietf-saa#8]
I suspect that it would be worth specifically addressing the line between "people" and "their ideas." This is likely to be contentious, but it's already come up more than once.

For example, which of the following statements would be considered unprofessional commentary? (all names are just examples)

* [ ] "Bob, your idea sucks, is stupid, and will kill the Internet."
* [ ] "Zoe's term as Area Director has been a disaster, and we'll be very happy when she's gone."
* [ ] "Bruce, your idea is flawed, because it doesn't consider the following situations..."
* [ ] "Joe works for FooCorp, and we know that they're Evil, so his idea should be rejected."
* [ ] "Only an idiot would come up with this idea."

Personally, I think the criticism of Bruce's idea is the only one above that's a professional message. That's because even though the rest attack an idea, they do so without actual valid criticism (typically, detailed arguments as to _why_ the idea is bad), so they are effectively using the idea as a proxy to attack the person. Nice hack, if you can get away with it.

**Comment by coopdanger** [comment]
I agree with your assessment of the examples, and between the language about insults and the description of disruptive behavior, I think the four inappropriate cases are covered by our definitions. I don't think the definitions should list examples, though, because no list of examples will be able to cover the waterfront of possible cases we might run into. I looked around at a lot of codes of conduct and did not find definitions more precise than the ones we came up with. Did you have some specific definition language in mind?

**Comment by elear** [comment]
Let's also be careful to circumscribe sarcasm a little bit here.  That is- the SAA should **not** get overly exercised over someone saying, “Yes, because doing <bad idea X> worked so well with <broken protocol Y>."

**Comment by mnot** [comment]
I think people have different things in mind when they read "Personal attacks include attacking people for their opinions, beliefs, or ideas rather than criticizing the opinions, beliefs, and ideas themselves." 

Something like "Criticising an idea unconstructively can amount to attacking the person" might help.

I disagree about examples; good specifications contain examples without implying that those are the only valid inputs, so I'm reasonably sure that we can include them without implying it's a complete list. Without concrete examples, I doubt we'll be able to get everyone on the same page here.

@elear your point about sarcasm would make a good counter-example :)

**Comment by ekr** [comment]
hmm... I'm actually not sure that I draw the line in the same place. For instance, taking the word "disaster" out of the equation, why is it inappropriate to say someone has been a bad AD? That's the kind of thing I hear all the time.

**Comment by mnot** [comment]
It's fine to say someone has been a bad AD if you give reasons that are unrelated to their person.

Saying someone is a bad AD because you don't like their face is unacceptable. Saying someone is a bad AD because they didn't show up is perfectly reasonable. Saying someone is a bad AD and not giving any more context, even by reference, leaves it ambiguous, and some will exploit that ambiguity.

**Comment by ekr** [comment]
I am sure some will exploit that ambiguity, but I don't think that ensuring that there is no ambiguity that could ever be exploited is the way that one ought to write rules of this sort.

**Comment by mnot** [comment]
The overall goal isn't to avoid ambiguity, it's to ensure professional behaviour. We already have a culture of requiring people to bring new information, rather than just +1'ing everything; this is just another expression of it. 

I.e., bring the relevant facts, just don't say "that's good" or "that's bad."

**Comment by ekr** [comment]
We don't have the SAA send people notes telling them not to +!.

**Comment by mnot** [comment]
If someone stands at the line in a meeting and says something like "Bob did a horrible job as editor on this spec", I'd encourage them to detail _why_. If they refused to, I'd take that as them trying to attack Bob rather than be constructive. 

Do we disagree on that?
